test/pspm_check_python_modules_test.m

```diff
@@ -0,0 +1,50 @@
+classdef pspm_check_python_modules_test < matlab.unittest.TestCase
+    % unittest class for the pspm_check_python_modules function
+    % PsPM TestEnvironment
+    % ● Authorship
+    % Abdul Wahab Madni 2024(Uni Bonn) 
+
+    methods (TestMethodSetup)
+        function addFunctionPath(testCase)
+            % Add the path to the source directory
+            src_path = fullfile(pwd, '..', 'src');
```

Comment:
In my test this does not seem to work.. I saw `~/GitHub/bachlab/PsPM/../src`. This path does not seem to be recognised by MATLAB.

Reply:
I don't think that will work because MATLAB does not directly recognize the tilde (~) as a shortcut for the home directory. We can try an indirect way like:
home_dir = getenv('USERPROFILE');  % This is for windows
if isempty(home_dir)
    % Fall back for macOS and Linux
    home_dir = getenv('HOME');
end

% Now construct the path with the determined home directory
src_path = fullfile(home_dir, 'GitHub', 'bachlab', 'PsPM', 'src');